support-cycle.md

```diff
@@ -0,0 +1,85 @@
+## Supported Cycle
+
+To ensure a consistent level of compatibility & support in our projects, each project under the Chai organisations must strictly adhere to a standardised lifecycle support policy, and platform support policy, as document below.
+
+### Supported Lifecycle
+
+We can only provide support for the latest version of any of our projects. As soon as a new release of any project is made, older versions of that project are considered end-of-life. We will not actively support older versions unless a negotiation with a third party has been made. Any actively supported older versions will be documented in the respective projects readme file.
+
+### Supported Platforms
+
+#### Node.js
+
+We support Node.js, following its LTS policy. [Node.js' release cycle is documented here](https://github.com/nodejs/LTS#release-schedule1) - which can be effectively summarised as: even numbered versions (4, 6, 8, 10) are supported from their date of release for around 3 years, while odd versions (5, 7, 9, 11) are supported for around 6 months. We follow this support cycle by supporting everything from the oldest release still in Maintainence to the current release. Any version that is marked *"End of Life"* we do not officially support, but we will continue to support any version marked as *"Maintainence"* or *"Active"*.
+
+##### Validating Node.js Support
+
+We validate our support for Node.js by running Continuous Integration testing our projects in the latest release of every major version - for example at the time of writing the current LTS release version is `6.11.13`; we will test this version but none of the earlier releases in the 6.0 branch.
+
+#### Firefox
+
+We support Firefox from it's _Extended Support Release_ (ESR) up until the _latest_ released version. [Firefox's ESR release cycle is documented here](https://www.mozilla.org/en-US/firefox/organizations/faq/) and can be effectively summarised as: a new ESR release is made approximately every year, while new regular releases are made every 6 weeks. For us this means we essentially support a years worth of Firefox releases.
+
+##### Validating Firefox Support
+
+We validate our support for Node.js by running Continuous Integration testing our projects in both the latest ESR of Firefox, as well as the latest version of Firefox. Any versions inbetween are not tested as it is unlikely that there will be failures in specific versions between the ESR and latest releases that aren't reflected in one or the other.
+
+#### Chrome
+
+Chrome has no Extended or Long Term Support cycle, and as such only the latest version so Chrome is officially supported by Google. We provide soft-support for Chrome releases up to a year old (that is, while we do not test any older releases, if a user finds a bug we will fix it).
```

Comment:
I think that this have a typo
Shouldn't it be `the latest version *of* Chrome` instead of `the latest version *so* Chrome`?